### PR TITLE
CLI Fixes #219 #220 #221

### LINKED ISSUE #219: CLI: Gracefully fail when version of `@misk` package can't be found
Currently ugly fails when old versions (ie. `0.1.1-alpha-0.1`) run `prebuild` on a new version of CLI.

Functionality should output a warning encouraging upgrade and not include the package in the `package.json`.

### LINKED ISSUE #221: Have cd prior concat with command for more consistent shelljs execution

### PR DESCRIPTION
**Pull request by adrw**
* [CLOSES #219] Gracefully fail when Misk Package of the miskTab.json version cannot be found.
* [#220] Use `lodash` instead of `lodash-es` because of errors when compiling using `tsc` vs `webpack`.
* [CLOSES #221] Fix prebuild with -e option by passing directory down through `prebuild` functions so that absolute paths can be used when manipulating and generating build files.

### CODE REVIEW

File: packages/@misk/cli/src/commands/prebuild.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
       throw new Error(`Migrating miskTab failed. ${e}`)
     })
     console.log("[PREBUILD] Generating up to date build files")
-    await generateBuildFiles().catch(e => {
+    await generateBuildFiles(...args).catch(e => {
       throw new Error(`Generating up to date build files failed. ${e}`)
     })
   } catch (e) {
-    console.log(`[ERROR] Generating up to date build files failed. ${e}`)
+    console.log(`[ERROR] Prebuild failed. ${e}`)
```
Comment (wesleyk) [inline packages/@misk/cli/src/commands/prebuild.ts]: why are we catching this exception and continuing without propagating it?

File: packages/@misk/cli/src/utils/generate.ts [hunk 2]
```diff
@@ -10,21 +10,33 @@ import {
...
 
-export const generateBuildFiles = async () => {
-  const pkg = await fs.readJson(Files.package)
-  const miskTab: IMiskTabJSON = await fs.readJson(Files.miskTab)
+export const generateBuildFiles = async (...args: any) => {
+  const dir = args[0]
+  let pkg = {}
+  try {
+    pkg = await fs.readJson(`${dir}/${Files.package}`)
+  } catch (e) {}
```
Comment (wesleyk) [inline packages/@misk/cli/src/utils/generate.ts]: ? Why are we ignoring exceptions here?

File: packages/@misk/cli/src/utils/index.ts [hunk 3]
```diff
@@ -41,10 +41,7 @@ export const prebuild = async (...args: any) =>
 
 export const execute = async (cmd: string, ...args: any) => {
   const dir = args[0]
-  if (dir) {
```
Comment (wesleyk) [inline packages/@misk/cli/src/utils/index.ts]: why did this change?

**Comment by wesleyk** [comment]
Yo, still looks like we're not splitting PRs out into granular, meaningful commits. Gonna go ahead and ask that you re-structure this. The three bullet points you listed above would make three perfect commits!

I ask for this because as a reviewer, particularly one new to the codebase and one unfamiliar with frontend, that would make it _worlds_ easier to review the code and ramp up on the codebase. It's a practice I'd like to see us get better at.

File: packages/@misk/cli/src/utils/generate.ts (hunk 2 above)
Comment (wesleyk) [reply packages/@misk/cli/src/utils/generate.ts]: synced up. We're using exception based control flow here, which is generally considered an anti-pattern. Let's find the proper "does file exist" API to use here that doesn't throw an exception if the file does not exist.

**Comment by adrw** [comment]
Merged with #229

**Pull request closed without merging**